connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -161,29 +155,19 @@ def __invoke_cosign(self, image, key):
 
         return process.returncode, stdout.decode("utf-8"), stderr.decode("utf-8")
 
-    @staticmethod
-    def __get_pubkey_config(key: str):
+    def __get_envs(self):
         """
-        Return a tuple of the used Cosign verification command (flag-value list), a
-        dict of potentially required environment variables and public key in binary
-        PEM format to be used as stdin to Cosign based on the format of the input
-        key (reference).
-
-        Raise InvalidFormatException if none of the supported patterns is matched.
+        Sets up environment variables used by cosign for potential authentication or TLS
```

Comment:
Docstring as command

is the use `for potential authentication or TLS verification against private registries.` relevant to the comment?